protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package sfx_log_format;
+
+option go_package = "gen_go/;log_format";
+
+// LogRequest represents the data that is incoming which is a list of ResourceLogs
+message LogRequest {
+    repeated ResourceLogs logs = 1;
+}
+
+// ResourceLogs handles either sharing a resource with list of LogRecord or unique for each logRecord
+message ResourceLogs {
+    Resource resource = 1;
+    repeated LogRecord logs = 2;
+}
+
+// model is based off https://github.com/open-telemetry/oteps/blob/master/text/0097-log-data-model.md#log-and-event-record-definition
+message LogRecord {
+    fixed64 Timestamp = 1;
+    bytes TraceID = 2;
+    bytes SpanID = 3;
+    fixed32 TraceFlags = 4;
```

Comment:
Since we only need one byte of this we can reserve the rest for future use. See how proposed it here: https://github.com/open-telemetry/opentelemetry-proto/pull/151/files

Up to you if you want to do the same. Not critical.